HLTrigger/Configuration/python/Tools/confdb.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from builtins import object
```

Comment:
Why is this single import needed - without any other change? Is it overriding a default?